x-pack/packages/kbn-ai-assistant/src/chat/chat_body.tsx

```diff
@@ -226,9 +227,36 @@ export function ChatBody({
   });
 
   const handleCopyConversation = () => {
+    const deserializeMessage = (message: Message): Message => {
```

Comment:
nitpick: We can probably define this `deserializeMessage` function outside of the scope of this component. It seems like a good candidate as an isolated function in the utils folder, as we might want to deserialize messages in other contexts too.

Reply:
(also then we don't recompile this function every time the component refreshes)

Reply:
Thanks for the review @sphilipse 
I've moved `deserializeMessage` to utils